cedar-policy-core/src/entities/json/schema_types.rs

```diff
@@ -319,6 +312,40 @@ pub struct HeterogeneousSetError {
     ty2: Box<SchemaType>,
 }
 
+/// Trying to compute the [`SchemaType`], but the value or expression
+/// contains an [`Unknown`] that has insufficient type information
+/// associated in order to compute the `SchemaType`
+#[derive(Debug, Diagnostic, Error)]
+pub struct UnknownInsufficientTypeInfoError {
+    /// `Unknown` which has insufficient type information
+    unknown: Option<Unknown>,
+}
+
+impl std::fmt::Display for UnknownInsufficientTypeInfoError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "cannot compute type because of insufficient type information for"
```

Comment:
I think you need a trailing space
```suggestion
            "cannot compute type because of insufficient type information for "
```